interfaces/cython/SConscript

```diff
@@ -137,6 +142,27 @@ elif not env["default_prefix"]:
     install_cmd.append(f"--prefix={env['prefix']}")
     python_prefix = env["prefix"]
 
+# Check for existing Python module installation. Allow pip to remove an existing
+# installation only we're installing to the same location. Also disable uninstallation
+# if we're installing to a staging directory.
+if env["stage_dir"]:
+    install_cmd.append("--ignore-installed")
```

Comment:
Would it make sense to combine this with the original content of the subsequent clause? (Line 166ff)

Reply:
I thought about doing that, but decided to keep them separate since the they are serving different purposes.